src/oc/plt.cpp

```diff
@@ -5,6 +5,8 @@
    */
 #include "hoc.h"
 #include "gui-redirect.h"
+
```

Comment:
just empty lines in this file

Reply:
They may seem empty, but they have fought valiant extern "C" battles ;)